src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx

```diff
@@ -168,3 +168,15 @@ Every pull request needs these labels so we can see where our contributions come
   - `from_external`: A user opened it in the repo OR it came in through #customer-feedback process.
   - `from_tw`: One of us created it (unless we were passing along #customer-feedback). If the PR fixes an external issue, label it as `from_tw` since the work was done by a tech writer.
 
+## Docs issues [#docs-issues]
+
+Docs issues is a docs team program that enlists the help of product teams in solving documentation and product issues. The goal is to get product team expertise in solving issues filed by other relics and our users. Docs issues candidates are issues that would be difficult for a docs hero to chase down or would benefit from deep engineering expertise. Examples of good candidates are code examples, requirements sections, and NRQL queries.
+
+The workflow for a viable issue looks something like this:
+
+1. A user files an GitHub issue and the GitHub hero identifies it as a potential docs issue.
+2. The hero marks the issue with the `docs-issue-candidate` label.
+3. Once a week, a designated docs team member goes through and assesses all issues with the `docs-issue-candidate` label.
+4. All approved issues get marked as `docs-issue` and migrated to the customer feedback Jira project.
+5. The new jiras are eventually triaged by PMs and funneled to the correct project team.
```

Comment:
"jiras" s/be "Jiras"
Maybe reword to "After product managers triage these issues, they funnel the Jiras to the appropriate project team."